src/cli/lambdapi.ml

```diff
@@ -79,6 +79,8 @@ let decision_tree_cmd : Config.t -> (Path.t * string) -> unit =
     let sym =
       let sign = Compile.compile false mp in
       try fst (StrMap.find sym Timed.(!(sign.sign_symbols)))
+      with Not_found ->
+      try fst (StrMap.find sym Timed.(Unif_rule.(!(sign.sign_symbols))))
```

Comment:
Why do we need this if ghost modules are open?

Reply:
Good remark, I'll look more into details.

Reply:
It's ok now that `find_sym` is used on a `sig_state`.